src/test/utilities/fakeAwsContext.ts

```diff
@@ -65,9 +66,19 @@ export class FakeAwsContext implements AwsContext {
     public onDidChangeContext: vscode.Event<ContextChangeEventsArgs> =
         new vscode.EventEmitter<ContextChangeEventsArgs>().event
     private awsContextCredentials: AwsContextCredentials | undefined
+    public lastTouchedRegion: string
```

Comment:
tip: you can initialize fields in-line
```suggestion
    public lastTouchedRegion = 'None'
```

Faster to read.